src/components/ControlBar.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
 import * as React from 'react';
-import { connect } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
Everything looks pretty great, I just commented team specific code style things

Reply:
Thanks so much for taking your time to review, Chris! I'll make the changes in a new branch to better match the team specific styles and to adopt it moving forward.